### PR TITLE
Solidstart-feature: counter component

### PR DESCRIPTION
**Pull request by Megio**
Added Counter Component, with tests and storybook suite.

Screenshot 2023-01-30 at 13 25 44

Screenshot 2023-01-30 at 13 25 29

Fixes: #887

### CODE REVIEW

**Review by hdJerry — CHANGES_REQUESTED** [review]
Everything is good... 
Can you please attach a screenshot of the UI both dev and storybook.

**Comment by Megio** [comment]
> Everything is good... Can you please attach a screenshot of the UI both dev and storybook.

Done!

File: starters/solidstart-tanstackquery-tailwind-modules/src/components/CounterExample/Counter.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { createSignal } from "solid-js";
+import { ActionButtonsData, CounterData } from "./types";
+
+const getCounterData = (): CounterData => {
+  const [count, setCount] = createSignal<number>(0);
```
Comment (hdJerry) [inline starters/solidstart-tanstackquery-tailwind-modules/src/components/CounterExample/Counter.tsx:5]: - The counter should be controlled by the global state manager
As it stands it is not global. You can check the other solidjs project to see how we did it. But I think you could just move this outside the function.

**Review by hdJerry — CHANGES_REQUESTED** [review]

**Review by hdJerry — APPROVED** [review]
Thanks Captain Mattia /Mattai/

**Pull request merged**